app/uk/gov/hmrc/cataloguefrontend/whatsrunningwhere/model.scala

```diff
@@ -23,6 +23,11 @@ import play.api.libs.json._
 import uk.gov.hmrc.cataloguefrontend.connector.model.{TeamName, Version}
 import uk.gov.hmrc.cataloguefrontend.model.Environment
 
+sealed trait Platform {
```

Comment:
Is this being used anywhere?

Reply:
Possibly not, this was in a conflict with master which I resolved. I could remove it and see if it compiles?